core/pom.xml

```diff
@@ -135,6 +135,12 @@
       <groupId>org.apache.xmlgraphics</groupId>
       <artifactId>fop</artifactId>
     </dependency>
+
+    <dependency>
+      <groupId>org.apache.xmlgraphics</groupId>
```

Comment:
Is this dependency necessary also in the main pom.xml?

Reply:
I think so, the root pom contains the version and this module the explicit dependency. 

We can remove from root and add here the version, but I would keep as it is in case the dependency can be required by other module(s).